apps/condo/domains/ticket/schema/TicketLastTimeViewedService.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Generated by `createservice ticket.TicketLastTimeViewedService`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/ticket/access/TicketLastTimeViewedService')
+const REDIS_URL = require('@core/config').REDIS_URL
+const IORedis = require('ioredis')
+const { get } = require('lodash')
+
+const dataSource = new IORedis(REDIS_URL)
+const TICKET_LAST_TIME_VIEWED_BY_USER_PREFIX = 'ticketLastTimeViewedByUser-'
+
+const TicketLastTimeViewedService = new GQLCustomSchema('TicketLastTimeViewedService', {
+    types: [
+        {
+            access: true,
+            type: 'input SetTicketLastTimeViewedInput { viewedAt: Float! }',
+        },
+        {
+            access: true,
+            type: 'type SetTicketLastTimeViewedOutput { status: String! }',
+        },
+        {
+            access: true,
+            type: 'type GetTicketLastTimeViewedOutput { syncedTicketLastTimeViewed: Float! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canTicketLastTimeViewed,
+            schema: 'setTicketLastTimeViewed(data: SetTicketLastTimeViewedInput!): SetTicketLastTimeViewedOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const viewedAt = Number(get(args, ['data', 'viewedAt']))
+                if (Number.isNaN(viewedAt) || viewedAt < 0 || !Number.isFinite(viewedAt) || !Number.isInteger(viewedAt)) {
+                    return {
+                        status: 'Wrong date',
+                    }
+                }
+                dataSource.set(TICKET_LAST_TIME_VIEWED_BY_USER_PREFIX + context.authedItem.id, viewedAt)
```

Comment:
Redis is not a good choice to store persistent data.

Reply:
Are Redis instances getting dropped between releases?